data/tlg0358/tlg001/__cts__.xml

```diff
@@ -1,6 +1,6 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0358" xml:lang="grc" urn="urn:cts:greekLit:tlg0358.tlg001">
   <ti:title xml:lang="lat">Introductio arithmetica</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0358.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0358.tlg001">
+  <ti:edition  xml:lang="eng"  urn="urn:cts:greekLit:tlg0358.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0358.tlg001">
```

Comment:
Since this is an edition, there does not need to be an `xml:lang` attribute. Go ahead and delete that.